mgmt/api/NetworkMessage.h

```diff
@@ -61,9 +61,9 @@ typedef enum {
   RECORD_DESCRIBE_CONFIG,
   LIFECYCLE_MESSAGE,
   UNDEFINED_OP /* This must be last */
-} OpType;
+};
 
-#define MGMT_OPERATION_TYPE_MAX (UNDEFINED_OP)
+#define MGMT_OPERATION_TYPE_MAX static_cast<unsigned>(OpType::UNDEFINED_OP)
```

Comment:
Changed it to a static_assert as per amc's recommendation. Then bike shedding ensued, so ignoring him now.